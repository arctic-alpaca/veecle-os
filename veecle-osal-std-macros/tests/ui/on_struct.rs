#[veecle_osal_std_macros::main]
struct Foo;
