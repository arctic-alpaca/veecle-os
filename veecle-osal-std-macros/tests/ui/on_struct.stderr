error: expected `fn`
 --> tests/ui/on_struct.rs:2:1
  |
2 | struct Foo;
  | ^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/on_struct.rs:2:12
  |
2 | struct Foo;
  |            ^ consider adding a `main` function to `$DIR/tests/ui/on_struct.rs`
